routes/web.php

```diff
@@ -14,5 +14,10 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
+    // List videos in a given chanel, return an array of PHP objects
+    $videoList = Youtube::listChannelVideos(env('YOUTUBE_CHANNEL_CODE'), 4);
```

Comment:
We shouldn't use `env()` directly in our code, as it's a performance issue; instead, set up a `config('services.youtube')` and read the `.env` in there